lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -218,17 +222,66 @@ type FlushingReader struct {
 	io.Reader
 	io.WriterTo
 
-	flusher GenericFlusher
+	flusher   GenericFlusher
+	initBytes []byte
 }
 
 func NewFlushingReader(rdr io.Reader) *FlushingReader {
-	return &FlushingReader{Reader: rdr, flusher: nil}
+	return &FlushingReader{Reader: rdr, flusher: nil, initBytes: nil}
+}
+
+func NewFlushingReaderWithInitBytes(rdr io.Reader, initBytes []byte) *FlushingReader {
+	return &FlushingReader{Reader: rdr, flusher: nil, initBytes: initBytes}
 }
 
 func (d *FlushingReader) AddFlusher(flusher GenericFlusher) {
 	d.flusher = flusher
 }
 
+// readDetectInit() is used by WriteTo() which is used by io.Copy.  It attempts
+// to detect a init byte buffer.  If it finds that init byte sequence, it is
+// ignored.  This reader does not care about the init sequeunce.  The init sequence
+// maybe used by the higher level interaction, which in this case is the Swagger
+// establishing initial connection for stdin.
+func (d *FlushingReader) readDetectInit(buf []byte) (int, error) {
+	nr, er := d.Read(buf)
+	if d.initBytes != nil {
+		initLen := len(d.initBytes)
+		total := nr
+		for {
+			if total < initLen {
+				if er != nil {
+					// Got an error, zero out len and break out to return error back
+					nr = 0
+					break
+				}
+				nr, er = d.Read(buf[total:])
+				total += nr
```

Comment:
Added code to kick out of monitoring the primer/init bytes if it doesn't look like we're getting them.